tests/test_layer.py

```diff
@@ -570,26 +570,26 @@ def helper_test_export_all_test_points(layer):
         test_points_file = os.path.join(temp_dir, "test_points.csv")
 
         try:
-            result = layer.export_all_test_points(
+            layer.export_all_test_points(
```

Comment:
Changed order to test the failure case first to ensure the exception type is set.
That way the next test can catch the custom exception instead of Exception and simplify formatting the failure message.